src/components/Table.js

```diff
@@ -1,74 +1,114 @@
 import React from "react";
-import GreenMark from "./StatusMarks/GreenMark";
-import RedMark from "./StatusMarks/RedMark";
-import NeutralMark from "./StatusMarks/NeutralMark";
+import moment from "moment";
+import "moment/locale/ru";
+
+import HabitStatus from "./HabitStatus";
+
+import { DateUtils } from "../utils/date";
+
+moment.locale("ru");
 
 const Status = {
   DONE: "DONE",
   FAILED: "FAILED",
+  NEUTRAL: "NEUTRAL",
   NOT_SPECIFIED: "NOT_SPECIFIED"
 };
 
-export default function Table({ HABITS }) {
-  function getMonday(d) {
-    d = new Date(d);
-    let day = d.getDay(),
-      diff = d.getDate() - day + (day === 0 ? -6 : 1); // adjust when day is sunday
-    return new Date(d.setDate(diff));
+export default class Table extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      READING_HABIT: {
+        name: "Чтение",
+        measuringValue: "Каждый день",
+        stats: [
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: Status.FAILED },
+          { status: Status.DONE },
+          { status: Status.FAILED },
+          { status: "" },
+          { status: "" }
```

Comment:
I would recommend keeping the habit status structure consistent - if you decided to use 4 states (3 real values and one for not specified status) then it is better to stick with them. 

Because when you do not provide status and use string literal `""` instead it makes other developer to think and raise unnecessary questions about why you decided to do that

Reply:
I forgot to change these statuses. Now it's fixed, thank you.